### PR TITLE
Shorter README with links to separate files

### PR DESCRIPTION
**Pull request by fxgst**
* quickstart guide
* shorter intro
* longform text, contributing and detailed install guide moved to other files (it's all still there)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,219 +1,41 @@
 # PocketIC: A Canister Testing Platform
 
-PocketIC is a local testing platform for canisters on the [Internet Computer](https://internetcomputer.org/).
+PocketIC is a local canister testing platform for the [Internet Computer](https://internetcomputer.org/).  
+It consists of the PocketIC **binary**, which handles the different IC instances, and a **Python library** (this repo) which allows you to interact with your canisters and the IC itself.
```
Comment (michael-weigelt) [inline README.md:4]:
```suggestion
It consists of the PocketIC **binary**, which handles different IC instances, and a **Python library** (this repo) which allows you to interact with your canisters and the IC itself.
```

File: README.md [hunk 2]
```diff
@@ -1,219 +1,41 @@
...
 
-
-Using PocketIC requires two components: A server and an integration library. 
-
-The **server** is compiled as part of the build process for the Internet Computer. It runs as local HTTP server which manages IC instances. Every test may request its own, _independent_ IC instance and interact with it concurrently with other running tests. 
-
-This PocketIC **library** provides a convenient Python frontend to interact with IC instances, while hiding the REST-API layer, the process lifetime etc. The test author simply interacts with either an IC instance:
-
-```python
```
Comment (michael-weigelt) [inline README.md:12]: How about about keeping both code examples; but shortening the prose text?

Comment (fxgst) [reply README.md:12]: Deal ;)

File: README.md [hunk 3]
```diff
@@ -1,219 +1,41 @@
...
-assert(counter_canister.read() == 1) 
+# call your canister functions with native Python syntax
+respone = my_canister.some_canister_function()
+assert(response == 'Hello, World!')
 ```
 
-If you need help with candid-encoding your `init_args`, canister call arguments and responses, check out the community developed [Python-agent](https://github.com/rocklabs-io/ic-py), which offers some useful candid functionality.
+## Getting Started
 
+### Quickstart
```
Comment (michael-weigelt) [inline README.md:21]: One of these titles is redundant

File: README.md [hunk 4]
```diff
@@ -1,219 +1,41 @@
...
-### Using pytest
-Coming soon
+For a more detailed installation guide, see [INSTALLATION.md](INSTALLATION.md).
 
-## Contributing
+### Examples
 
-First install [Nix](https://nixos.org/download.html) then bring the development
-environment into scope using either:
+To see some working code, see the [examples](examples/) folder, or check out the [how to use this library](HOWTO.md) guide.
```
Comment (michael-weigelt) [inline README.md:31]: how about mentioning the howto first (as it is an introduction) and then the "real" production-ready ;) example?

File: README.md [hunk 5]
```diff
@@ -1,219 +1,41 @@
...
+* [Why PocketIC](WHY.md)
+* [How to use this library](HOWTO.md)
 
-Run tests using:
 
-```
-$ poetry run pytest
-```
\ No newline at end of file
+## Contributing to PocketIC
```
Comment (michael-weigelt) [inline README.md:39]: technically, they would contribute to pocketic-py, or pocket_ic, not the "whole" PocketIC

File: README.md [hunk 6]
```diff
@@ -1,219 +1,41 @@
...
+assert(response == 'Hello, World!')
 ```
 
-If you need help with candid-encoding your `init_args`, canister call arguments and responses, check out the community developed [Python-agent](https://github.com/rocklabs-io/ic-py), which offers some useful candid functionality.
+## Getting Started
 
+### Quickstart
+* Download the **PocketIC binary** for [Linux](https://download.dfinity.systems/ic/71adba179679a1090baa26cadcceadec311f57b3/openssl-static-binaries/x86_64-linux/pocket-ic.gz) or [macOS](https://download.dfinity.systems/ic/71adba179679a1090baa26cadcceadec311f57b3/openssl-static-binaries/x86_64-darwin/pocket-ic.gz).
+* Make sure the binary is available through your `$PATH` variable; you can verify that everything works by calling `pocket-ic --help` from your terminal.
+* Run `pip3 install pocketic-py` to get the Python library.
```
Comment (michael-weigelt) [inline README.md:24]: maybe add: "from your virtual environment"? 

I don't like suggesting a bad practice. We don't know the expertise of our audience.

Comment (michael-weigelt) [inline README.md:24]: the package is called pocket_ic, so they should get that from PyPi. pocketic-py is the name of the repo.

File: README.md [hunk 7]
```diff
@@ -1,219 +1,41 @@
 # PocketIC: A Canister Testing Platform
 
-PocketIC is a local testing platform for canisters on the [Internet Computer](https://internetcomputer.org/).
+PocketIC is a local canister testing platform for the [Internet Computer](https://internetcomputer.org/).  
+It consists of the PocketIC **binary**, which handles different IC instances, and a **Python library** (this repo) which allows you to interact with your canisters and the IC itself.
```
Comment (michael-weigelt) [inline README.md:4]:
```suggestion
It consists of the PocketIC **binary**, which handles different IC instances, and a **Python library** (this repo) which allows you to interact with your local IC instances and the canisters thereon. 
```

File: README.md (hunk 6 above)
Comment (fxgst) [reply README.md:24]: I'm not sure, for ic-py it's also like this: `pip install ic-py`, but then `import ic`

Comment (michael-weigelt) [reply README.md:24]: ok I guess we'll see when we have published it. But I would want to make sure that the name of our published package is just pocket_ic

File: README.md (hunk 4 above)
Comment (fxgst) [reply README.md:31]: But the section is called examples, and for a quick start you maybe just want to look at examples first. How to use this library is more documentation and a longer read

**Review by michael-weigelt — APPROVED** [review]

**Pull request merged**